sign_oca/models/sign_oca_request.py

```diff
@@ -25,19 +25,20 @@ class SignOcaRequest(models.Model):
     _name = "sign.oca.request"
     _inherit = ["mail.thread", "mail.activity.mixin"]
     _description = "Sign Request"
+    _order = "state, create_date desc, id desc"
```

Comment:
The other field doesn't add value

```suggestion
    _order = "state, id desc"
```